src/ds5/ds5-auto-calibration.cpp

```diff
@@ -850,7 +885,7 @@ namespace librealsense
 
         if (depth > 0)
         {
-            LOG_DEBUG("run_tare_calibration interactive control with parameters: depth = " << depth);
+            LOG_OCC_DEBUG("run_tare_calibration interactive control (2) with parameters: depth = " << depth);
```

Comment:
Should this "(2)" remain in the log text?

Reply:
Yes,  - for debug purposes